conf/flight_plans/tudelft/imav2025.xml

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE flight_plan SYSTEM "../flight_plan.dtd">
+
+<flight_plan alt="2150" ground_alt="2080" lat0="18.9426688" lon0="-98.1893464" max_dist_from_home="1500" name="Basic" security_height="2130">
```

Comment:
Except for the ground alt, what is the difference with the basic flight plan ? Is it really necessary to keep it?